packages/web-helpers/src/WebUser.ts

```diff
@@ -1,11 +1,11 @@
-import { Client, Network, Context } from '@verida/client-ts';
+import { Client, Network, type Context } from '@verida/client-ts';
 import { VaultAccount, hasSession } from '@verida/account-web-vault';
 import { EventEmitter } from 'events'
-import { IProfile, ClientConfig, ContextConfig, MessageSendConfig, IDatastore, IDatabase, DatabaseOpenConfig, DatastoreOpenConfig, AccountVaultConfig } from '@verida/types';
+import type { IProfile, ClientConfig, ContextConfig, MessageSendConfig, DatabaseOpenConfig, DatastoreOpenConfig, AccountVaultConfig } from '@verida/types';
 
 export interface WebUserProfile {
     name?: string
-    avatarUri: string
+    avatarUri?: string
```

Comment:
This is not a required field in the profile schema

Reply:
Yes, that's why I put it optional here, it was not.